src/js/game/Entities/Player.js

```diff
@@ -4,7 +4,7 @@ const CallbackCommand = require("../CommandQueue/CallbackCommand.js");
 module.exports = class Player extends BaseEntity {
   constructor(controller, type, x, y, name, isOnBlock, facing) {
     super(controller, type, 'Player', x, y, facing);
-    this.offset = (name === 'Agent') ? [-5, -15] : [-18, -32];
+    this.offset = (name === 'Player') ? [-5, -15] : [-18, -32];
```

Comment:
This will set the Player offset to `[-5, -15]` which isn't correct.

Reply:
is [-18, -32] the desired offset? When is the name anything other than Player? I didn't see the need for this offset shuffle, but perhaps I just haven't seen where it comes into play?

Reply:
I could be mistaken -- I think [-18, -32] is correct for the player. That was the value before I added [-5, -15] for the Agent.